incremental-font-transfer/src/glyph_keyed.rs

```diff
@@ -500,9 +526,12 @@ trait GlyphDataOffsetArray {
         font_builder: &mut FontBuilder<'a>,
         data: impl Into<Cow<'a, [u8]>>,
         offsets: impl Into<Cow<'a, [u8]>>,
+        offset_type: OffsetType,
     ) -> Result<(), PatchingError>;
 }
 
+const LOCA_OFFSET_TYPES: [OffsetType; 2] = [OffsetType::ShortDivByTwo, OffsetType::Long];
```

Comment:
I'd make this an associated constant on `OffsetType`? (Are we expecting more offset types?)

Reply:
since it's only needed inside the available_offset_types() method I moved it into there.